amgut/db/scrub.sql

```diff
@@ -0,0 +1,343 @@
+-- This script is used to scrub the live American Gut database in order to
+-- generate a test database for Travis without personal information but
+-- representative of the real world.
+
+-- Function to generate a string with all the different characters present in
+-- a column (adapted from: http://dba.stackexchange.com/a/83679)
+CREATE FUNCTION retrieve_chars(column_name varchar, table_name varchar) RETURNS varchar AS $$
```

Comment:
I'd feel substantially better about this approach if it the result was always assured to include `[a-z]` to handle possible low entropy responses.

Given that the goal is to ensure proper handling of non-ascii characters, is this method suitable and tested to be suitable for non-ascii? The SE response says so, but given the nature of this PR, it would be good to have added confidence. 

I'm not sure to the extent we need to be concerned, but UTF-8 is not the only character set we encounter, so it may be worth asserting separately that this method can handle other common ones like UTF-16, ISO-8859-1, and ANSI